### PR TITLE
hello microverse

### PR DESCRIPTION
**Pull request by BilalLiaquat7**
In this PR: I have done the following things

- Add Linters
- Add HTML 
- Add CSS file
- Add README file
- updated README file
- Add git ignore file
- Add test file and added it in gitignore
- Add node modules folder and added it in gitignore
- Added README content (main section)

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello  @BilalLiaquat7     , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlinters
- Correct git-flow :heavy_check_mark: 
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly make sure to update your PR description by proving a well-detailed summary that makes your PR clean and professional and even self-explanatory. :wink: 

   - Example of good PR Summary
image

  ### HOW TO EDIT PR DESCRIPTION
   chrome-capture-2023-3-27
 
- [x] Kindly make sure to include all main sections from the [Microverse README template](https://github.com/microverseinc/readme-template) which will help to explain your repository contents and have a clean and professional Readme. Please make sure to customize it according to your project and fill in the needed information and remember all main sections must be included except the FAQ section which is optional.
 
   **_I can see that you have tried to use the template but you removed the important sections. Kindly make sure all sections are presented and make sure to remove all comments and placeholders and add your project information_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure to write your commit message in the imperative tense and capitalize it. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (Alusp) [inline README.md:1]: - [x] [Required] A very good job so far, however, As requested by the previous Code Reviewer, Please do not Remove the main sections from the Readme template, It serves as a documentation tool that provides important information about the project and helps users, contributors, and maintainers understand and work with the codebase effectively. Please use this [template](https://github.com/microverseinc/readme-template) and edit it to suit your Hello Microverse project. :smile:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @BilalLiaquat7 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost there


 ## Highlights

- [ ] Correct git-flow :white_check_mark: 
- [ ] No Linter Errors :white_check_mark: 
- [ ] Good commit history :white_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

- [x] [Required] As requested by the Previous Reviewer, Please be very descriptive in your PR, a well-written and descriptive PR promotes effective communication, collaboration, and understanding among team members. It streamlines the review process, facilitates onboarding, improves code maintenance, and serves as a valuable source of documentation and knowledge sharing. Investing time and effort in creating informative PRs can significantly benefit the development workflow and contribute to the overall success of a project. :smile: 
- Example of a descriptive PR below. :arrow_down: 
image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @BilalLiaquat7  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**